com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs

```diff
@@ -527,6 +559,56 @@ static class LitKeywords
                 { CoreKeywordDescriptors.ShadowsSoft },
                 { CoreKeywordDescriptors.MixedLightingSubtractive },
             };
+            
+            public static KeywordCollection DataExtraction = new KeywordCollection
+            {
+                new KeywordDescriptor()
+                {
+                    displayName = "Extraction Modes",
+                    referenceName = "",
+                    type = KeywordType.MultiCompile,
+                    definition = KeywordDefinition.MultiCompile,
+                    scope = KeywordScope.Global,
+                    value = 0,
+                    entries = new[]
+                    {
+                        new KeywordEntry("None", "_"),
+                        new KeywordEntry("Object Id", "RENDER_OBJECT_ID"),
+                        new KeywordEntry("Depth", "RENDER_DEPTH"),
+                        new KeywordEntry("World Normals (face)", "RENDER_WORLD_NORMALS_FACE"),
+                        new KeywordEntry("World Normals (pixel)", "RENDER_WORLD_NORMALS_PIXEL"),
+                        new KeywordEntry("World Position", "RENDER_WORLD_POSITION"),
+                        new KeywordEntry("Base Color + Alpha", "RENDER_BASE_COLOR_ALPHA"),
+                        new KeywordEntry("Specular(RGB) Metallic(A)", "RENDER_SPECULAR_METALLIC"),
+                        new KeywordEntry("Emission(RGB)", "RENDER_EMISSION"),
+                        new KeywordEntry("Smoothness(r) + Occlusion (g)", "RENDER_SMOOTHNESS_OCCLUSION"),
+                        new KeywordEntry("EntityId", "RENDER_ENTITY_ID"),
```

Comment:
each entry should have `RENDER_` removed from the reference name field, as the generator already formats these as needed for multicompile enum entries